README.md

```diff
@@ -2,6 +2,93 @@
 
 Build images from the commandline in a convenient way.
 
+## Installation
+
+This project is under development right now and needs to be run via:
+```
+$ go run github.com/osbuild/image-builder-cli/cmd/image-builder@main
+```
+we plan to provide rpm packages as well.
+
+
+## Prerequisites
+
+Make sure to have the required `osbuild` rpms installed:
+```
+$ dnf install osbuild osbuild-depsolve-dnf
+```
+for the repositories you need `osbuild-composer` as well (this will
+change).
+
+## Example
+
+To see the list of buildable images run:
+```
+$ ./image-builder list-images
```

Comment:
Is the subcommand `list-images`, or `list-image`? The examples are not consistent. Which form is the canonical one?

Reply:
It is "list-images". I also fixed that now.